.travis.yml

```diff
@@ -54,6 +54,28 @@ matrix:
       env:
         - MATRIX_EVAL="CXX=g++-7"
 
+    # Lint Python 2 with flake8
+    - os: linux
+      language: python
+      python: 2.7
+      before_install: pip install flake8
+      script:
+        # stop the build if there are Python 2 syntax errors or undefined names
+        - flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
+        # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
+        - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+
+    # Lint Python 3 with flake8
+    - os: linux
```

Comment:
Could you give this job a name (such as `Python linting`)? If you do so, please add it to the other Python job as well (the autoformat one).